changelog.md

```diff
@@ -1,3 +1,121 @@
+
+# CITYENGINE SDK 3.3.11173 CHANGELOG
+
+This section lists changes compared to CityEngine SDK 3.2.10824.
+
+## General Info
+* CityEngine SDK 3.3.11173 is used in CityEngine 2025.0.11173.
+
+## PRT API
+* `prt::InitalShapeBuilder::setAttributes()`: support for disabling the numerical geometry cleanup .
+* `prt::generate()`: Fixed a small mem leak when CGA imports were used.
```

Comment:
why abbreviate? "memory" instead of "mem"

Reply:
memory leak sounds totally uncool. but I went for it :)